StudentProject/StudentDetails.java

```diff
@@ -48,88 +27,86 @@ public static void main(String[] args) {
             ch = s.nextInt();
 
             switch (ch) {
-                case 1:
+                case ADD_STUDENT:
                     System.out.println("--------------------------------------------------");
                     System.out.println("Please provide the data for the student to store. ");
                     System.out.print("Student id: ");
-                    int sno = s.nextInt();
+                    int eno = s.nextInt();
```

Comment:
What is _eno_? Use meaningful names, if that is the student id, then use the name _studentId_ instead of _eno_.